src/routes/(app)/order/[id]/payment/[paymentId]/ticket/+page.server.ts

```diff
@@ -0,0 +1,34 @@
+import { error } from '@sveltejs/kit';
+import { runtimeConfig } from '$lib/server/runtime-config.js';
+import { fetchOrderForUser } from '../../../fetchOrderForUser.js';
+
+export async function load({ params }) {
+	const order = await fetchOrderForUser(params.id);
+	if (!order) {
+		throw error(404, 'Order not found');
+	}
+	const payment = order.payments.find((payment) => payment.id === params.paymentId);
+	if (!payment) {
+		throw error(404, 'Payment not found');
+	}
+	if (payment.status !== 'paid') {
+		if (payment.invoice?.number) {
+			throw error(400, 'Invoice already created on pending payment');
+		}
+	} else {
+		if (!payment.invoice?.number) {
+			throw error(400, 'Invoice not found');
+		}
+	}
+
+	if (!runtimeConfig.sellerIdentity && !order.sellerIdentity) {
```

Comment:
Since you're using the converse of this predicate in the return statement, it's preferable to make a variable.

```suggestion
    const sellerIdentity = runtimeConfig.sellerIdentity || order.sellerIdentity;
	if (!sellerIdentity) {
```

Then use it in the return statement.